packages/fleather/lib/src/rendering/editable_text_line.dart

```diff
@@ -285,18 +285,10 @@ class RenderEditableTextLine extends RenderEditableBox {
   // Computes the box height for the position.
   // To get the correct box, it extends position backward if position
   // is bigger than 0 and extends it forward if it's text position is 0.
-  double _getBoxHeightForPosition(TextPosition position) {
-    final int baseOffset;
-    if (position.offset > 0) {
-      baseOffset = position.offset - 1;
-    } else {
-      baseOffset = position.offset;
-    }
+  double _getLineHeightForPosition(TextPosition position) {
+    final lineBoundary = getLineBoundary(position);
     final boxes = body!.getBoxesForSelection(TextSelection(
-      baseOffset: baseOffset,
-      extentOffset: baseOffset + 1,
-      affinity: position.affinity,
-    ));
+        baseOffset: lineBoundary.start, extentOffset: lineBoundary.end));
     return boxes.firstOrNull?.toRect().height ?? 0;
```

Comment:
Don't we rather want to get the max height of all boxes of the line?

Reply:
Based on my tests it returns one box but that's better. I will change it now.